modules/wabt/1.0.37/overlay/include/wabt/BUILD.bazel

```diff
@@ -0,0 +1,15 @@
+load("@rules_cc//cc:cc_library.bzl", "cc_library")
+load(":config_h.bzl", "config_h")
+
+config_h(name = "config_h")
+
+cc_library(
+    name = "wabt",
+    hdrs = glob([
+        "*.h",
+        "interp/*.h",
+    ]) + [":config_h"] + glob(["*.def"]),
```

Comment:
Can the two glob calls be merged into one? Do the `*.def` files need to be in here at all, in addition to `textual_hdrs`?

Reply:
Since `hdrs` leaks this to dependents, would it be possible to move the textual headers to a separate target in `implementation_deps`?